bindings/wasm/bindings.ts

```diff
@@ -0,0 +1,43 @@
+type Matrix1x3 = [Number, Number, Number];
+type Matrix4x3 = [Matrix1x3, Matrix1x3, Matrix1x3, Matrix1x3];
+type Vec3 = [Number, Number, Number];
+type Vec3Fun =
+    ((v: Vec3) => Manifold)&((x: Number, y?: Number, z?: Number) => Manifold);
```

Comment:
I don't quite understand this; what does this do that a `Vec3` input would not?

Reply:
It seems that I remembered the wrong thing, I somehow thought that OpenSCAD allows both `translate(x, y, z)` and `translate([x, y, z])`, and this is used to allow this. Will remove this.